README.md

```diff
@@ -305,6 +305,70 @@ Run the tool through the context menu on a file or directory:
 
 </details>
 
+### Workflow for GitHub Actions
+
+<details>
+
+#### What does it do?
+
+This workflow will convert every JPG, JPEG and PNG file into AVIF and WebP.
+
+#### Add Workflow
+
+Add the following file in the following location:
+`.github/workflows/optimizt.yml`
+
+##### Push changes trough commit
+
+The current workflow will push changes based on commit. This means when anything happens with a JPG/JPEG/PNG it will be triggerd.
+
+Insert the following into optimizt.yml if you want this process fully automatic.
+```yml
+name: optimizt
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch and only when there's JPG/JPEG/PNG in the commmit!
+  push:
+    branches: [main]
+    paths:
+      - "**.jpe?g"
+      - "**.png"
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+jobs:
+  run-optimizt:
+    runs-on: ubuntu-latest
+    env:
+      OPTIMIZTCONVERTERARGS: --verbose --force --avif --webp . # convert to avif and webp for all JPG/JPEG/PNG files in this folder
```

Comment:
Let's split words in this variable using underscore? `OPTIMIZT_CONVERTER_ARGS`. Also I think `OPTIMIZT_ARGS` is shorter and more readable.

Reply:
By the way, why do we need this env variable here? It's used only once. Should not it be inlined in `run optimizt` section?